common/beerocks/scripts/prplmesh_utils.sh.in

```diff
@@ -163,13 +163,13 @@ stop_function() {
 }
 
 main_agent_operational() {
-    pgrep -la beerocks_agent | grep -v wlan > /dev/null 2>&1 || return 1
+    pgrep -fx "@BEEROCKS_INSTALL_PATH@/bin/beerocks_agent" > /dev/null 2>&1 || return 1
     grep -q 'CONNECTED --> OPERATIONAL' "$1/beerocks_agent.log"
 }
 
 radio_agent_operational() {
-    pgrep -la beerocks_agent | grep $2 | grep -v grep > /dev/null 2>&1 || return 1
-    grep -q 'CSA' "$1/beerocks_agent_$2.log"
+    pgrep -fx "@BEEROCKS_INSTALL_PATH@/bin/beerocks_agent -i $2" > /dev/null 2>&1 || return 1
+    grep -q 'goto STATE_OPERATIONAL' "$1/beerocks_agent_$2.log"
```

Comment:
I certainly agree that this change is needed (in fact, I though we had done this already a while ago), but it should be in a separate commit with a good commit message.